treeherder/perf/secretary_tool.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import simplejson as json
+
+from datetime import datetime, timedelta
+from treeherder.perf.models import PerformanceSettings
+from django.conf import settings as django_settings
+
+logger = logging.getLogger(__name__)
+
+# TODO: extract strings into constants
+
+
+class SecretaryTool:
+    """
+    Tool used for doing the secretary work in the Performance Sheriff Bot.
+    """
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def manage_settings():
+        perf_sheriff_settings, created = PerformanceSettings.objects.get_or_create(
+            name="perf_sheriff_bot",
+            defaults={"settings": SecretaryTool._get_default_settings()},
+        )
+
+        if created:
+            logger.info("Performance settings for perf_sheriff_bot not found. Creating with defaults...")
+
+        settings = json.loads(perf_sheriff_settings.settings)
+        # reset limits if the settings expired
+        if SecretaryTool.is_expired(datetime.fromisoformat(settings["last_reset_date"])):
```

Comment:
How about something like `cls.are_expired(settings)`?
The client code looks easier on the eyes, plus you hide some implementation details.